### ISSUE TITLE
Cannot install CARMA

### ISSUE THREAD

**Issue by sayangsep** [ZikunY/CARMA#24]
Hi,

I am facing an error during installation of the package. Could you please help me resolve it?

Thanks,
Sayan 

--

****Warning: /tmp/Rtmpp8HyId/R.INSTALL232e2706012f4/CARMA/man/CARMA.Rd:93: unexpected section header '\examples'
Warning: /tmp/Rtmpp8HyId/R.INSTALL232e2706012f4/CARMA/man/CARMA.Rd:113: unexpected END_OF_INPUT '
'
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
Error: package or namespace load failed for ‘CARMA’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/home/sayan/R/x86_64-pc-linux-gnu-library/4.4/00LOCK-CARMA/00new/CARMA/libs/CARMA.so':
  /home/sayan/R/x86_64-pc-linux-gnu-library/4.4/00LOCK-CARMA/00new/CARMA/libs/CARMA.so: undefined symbol: dsyevd_
Error: loading failed
Execution halted
ERROR: loading failed
removing ‘/home/sayan/R/x86_64-pc-linux-gnu-library/4.4/CARMA’
Warning message:
In i.p(...) :
  installation of package ‘/tmp/Rtmpvt3S2A/file221db770c6972/CARMA_1.0.tar.gz’ had non-zero exit status****

**Comment by ZikunY** [comment]
Hi Sayan,

 Thank you for your interest in CARMA. Before diving into the details, have you tried the solution on the webpage of this repository? "Intel MLK library" and "Implementation on M1 Mac".

**Comment by sayangsep** [comment]
It worked. Thanks a lot!

**Comment by dfermin** [comment]
In case anyone else encounters this issue on RHEL.

I installed the intel MLK packages after adding the Intel Repo
`sudo yum install intel-oneapi-mkl-devel intel-oneapi-mkl`

Then I declared this variable in bash:
```
export LD_PRELOAD=/opt/intel/oneapi/mkl/latest/lib/libmkl_rt.so
````
Then I could install CARMA.

Looks like the intel libraries are install in `/opt` by default and not in `/usr/lib` or /usr/lib64` which is where R looks.
